File: RQ2/ts/tests/cases/_relation-override/_subclass-overrides-method/file0.js
class Foo {
    method0() {
        console.log('Foo')
    }

    get a() {
        return 0;
    }
}

class Bar extends Foo {
    method0() {
        console.log('Bar')
    }

    get a() {
        return 1;
    }
}

new Bar().method0() // 'Bar'
new Bar().a // 1

File: RQ2/java/tests/cases/_class/_anonymous-class-declaration/file0.java
class Foo {
  
    interface Bar {
        void a();
    }
  
    public void doThings() {

        /**
         * Below creates 2 anonymous class which
         * implements interface `Bar` instantly
         */
        Bar anony0 = new Bar() {
            public void a() {
                /* ... */
            }
        };

        Bar anony1 = new Bar() {
            public void a() {
                /* ... */
            }
        };
        
    }         
}